src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Channel.xml

```diff
@@ -24,23 +24,28 @@
         <attribute name="name">
             <group>admin:channel:read</group>
             <group>admin:channel:create</group>
+            <group>admin:channel:update</group>
             <group>shop:channel:read</group>
         </attribute>
         <attribute name="enabled">
             <group>admin:channel:read</group>
             <group>admin:channel:create</group>
+            <group>admin:channel:update</group>
         </attribute>
         <attribute name="description">
             <group>admin:channel:read</group>
             <group>admin:channel:create</group>
+            <group>admin:channel:update</group>
         </attribute>
         <attribute name="hostname">
             <group>admin:channel:read</group>
             <group>admin:channel:create</group>
+            <group>admin:channel:update</group>
         </attribute>
         <attribute name="color">
             <group>admin:channel:read</group>
             <group>admin:channel:create</group>
+            <group>admin:channel:update</group>
         </attribute>
         <attribute name="baseCurrency">
```

Comment:
Shouldn't `baseCurrency` be exposed as well?

Reply:
[REF](https://github.com/Sylius/Sylius/pull/15452#discussion_r1368560892)